solr/core/src/test/org/apache/solr/cloud/ReplicationFactorTest.java

```diff
@@ -461,38 +447,24 @@ protected void doDelete(UpdateRequest req, String msg, int expectedRf, int retri
 
   protected int sendDoc(int docId, int minRf) throws Exception {
     UpdateRequest up = new UpdateRequest();
-    boolean minRfExplicit = maybeAddMinRfExplicitly(minRf, up);
     SolrInputDocument doc = new SolrInputDocument();
     doc.addField(id, String.valueOf(docId));
     doc.addField("a_t", "hello" + docId);
     up.add(doc);
-    return runAndGetAchievedRf(up, minRfExplicit, minRf);
+    return runAndGetAchievedRf(up, minRf);
   }
   
-  private int runAndGetAchievedRf(UpdateRequest up, boolean minRfExplicit, int minRf) throws SolrServerException, IOException {
+  private int runAndGetAchievedRf(UpdateRequest up, int minRf) throws SolrServerException, IOException {
     NamedList<Object> response = cloudClient.request(up);
-    if (minRfExplicit) {
-      assertMinRfInResponse(minRf, response);
-    }
     return cloudClient.getMinAchievedReplicationFactor(cloudClient.getDefaultCollection(), response);
   }
 
   private void assertMinRfInResponse(int minRf, NamedList<Object> response) {
-    Object minRfFromResponse = response.findRecursive("responseHeader", UpdateRequest.MIN_REPFACT);
+    Object minRfFromResponse = response.findRecursive("responseHeader");
     assertNotNull("Expected min_rf header in the response", minRfFromResponse);
     assertEquals("Unexpected min_rf in response", ((Integer)minRfFromResponse).intValue(), minRf);
   }
```

Comment:
Looks like `assertMinRfInResponse` is now also unused then, if so suggest to remove it also.